feature/system/gnmi/cliorigin/mixed_oc_cli_origin_support_test/README.md

```diff
@@ -0,0 +1,55 @@
+# gNMI-1.1: cli Origin
+
+## Summary
+
+Ensure that both CLI and OC configuration can be pushed to the device at the
+same time
+
+## Procedure
+
+Note: this test is intended to cover only the case of pushing some configuration
+along with OC paths - since it is unknown what CLI configuration would be
+required in the emergency case that is covered by this requirement.
+
+*   TODO: Push base configuration to DUT specifying an interface configuration
```

Comment:
Why is this TODO?

Reply:
We actually do not push base config in cliOrigin and mixed_cli_origin, which is why in both these tests this step is marked as TODO. We just push specific config relevant to that interface. This step can probably be removed for this test as its not required.